.gitignore

```diff
@@ -43,7 +43,7 @@ NashornProfile.txt
 **/JTreport/**
 **/JTwork/**
 /src/utils/LogCompilation/target/
-/.project/
+/.project
```

Comment:
The [upstream commit](https://github.com/ibmruntimes/openj9-openjdk-jdk/commit/66c80c80d2e0b1d64a77b8bed644c8c3f7506719) in question removed `/.project` and kept `/.project/`.
https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/599/commits/7480876ba34a057cc812d794331769bfc8c24fc2 removed `/.project` as well, what's the rationale to bring it back and remove `/.project/`?

Reply:
With the trailing slash an entry only matches directories while `.project` is expected to be a file (`/.project/` was probably never helpful). https://github.com/ibmruntimes/openj9-openjdk-jdk/commit/7480876ba34a057cc812d794331769bfc8c24fc2 removed one instance of `/.project` but left the other (from upstream).